### DISCUSSION TITLE
Seb client instalation

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-win-refactoring#913] [Q&A]
Hello I hope you are well.
I need help.
I have two students, who use Windows, they installed the latest version of SEB. When they download the connection file to enter the exam, this screen appears, from which they cannot exit:
![Seb problem](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/94466010/cb62d856-b1a9-4fc3-be81-406030a5fc94)
The file appears as seb file, not as seb configuration file.

I have choose "Open With" then choose Safe Exam Browser, and the problem continues.

Thank you very much! 

Melina Rapacioli

**Comment by dbuechel** [comment]
The system configuration for SEB configuration files is then most likely incorrect, you should be able to change this in the system settings (see "Choose default apps by file type" as explained under https://support.microsoft.com/en-us/windows/change-default-programs-in-windows-e5d82cad-17d1-c53b-3505-f10a32e1894d):

![grafik](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/28347927/7135e0de-b968-49d9-8a0b-00b5d5c1faa8)

You could also try to uninstall all SEB versions potentially found under "Apps" in the system settings, restart the machine and try to install the latest release version of SEB.
